src/debugtopus/index.ts

```diff
@@ -24,6 +24,7 @@ type DebugtopusOptions = {
   bypassProxy?: boolean;
   browser?: "CHROMIUM" | "FIREFOX" | "SAFARI";
   breakpoint?: "DESKTOP" | "MOBILE" | "TABLET";
+  onlyActivated?: boolean;
```

Comment:
I mean maybe that's not really a use case but shouldn't this just be `--run-status=ON` or `--run-status=OFF` ? a) more consistent with the other params and b) more flexible